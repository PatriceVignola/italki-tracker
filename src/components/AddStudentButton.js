/**
 * @prettier
 * @flow
 */

import React from 'react';
import Button from '@material-ui/core/Button';
import AddIcon from '@material-ui/icons/Add';

type Props = {
  onClick: () => void,
};

function AddStudentButton(props: Props) {
  return (
    <Button
      {...props}
      onClick={props.onClick}
      variant="fab"
      color="secondary"
      aria-label="add"
    >
      <AddIcon />
    </Button>
  );
}

export default AddStudentButton;
