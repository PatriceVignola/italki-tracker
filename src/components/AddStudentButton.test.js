/**
 * @prettier
 * @flow
 */

import React from 'react';
import {shallow} from 'enzyme';

import AddStudentButton from './AddStudentButton';

describe('AddStudentButton', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<AddStudentButton onClick={jest.fn()} />);
    expect(wrapper).toMatchSnapshot();
  });
});
