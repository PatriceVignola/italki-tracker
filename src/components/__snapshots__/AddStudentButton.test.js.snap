// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddStudentButton renders correctly 1`] = `
<WithStyles(Button)
  aria-label="add"
  color="secondary"
  onClick={[Function]}
  variant="fab"
>
  <pure(Add) />
</WithStyles(Button)>
`;
